### PR TITLE
Add object-to-JSON value conversion

### PR DESCRIPTION
**Pull request by tbroadley**
Hi there! I've been working on integration `ngrx-forms` with a custom form control called [`ngx-chips`](https://github.com/Gbuomprisco/ngx-chips), which uses an array as its value. I wrote a value converter that transforms the array into a string by calling `JSON.stringify` on it. I thought I'd check if this value converter is something you'd be interested in adding as a built-in to `ngrx-forms`.

Before this is merged, I'd like to add tests that check the conversion of strings, numbers, booleans, and arrays. Currently, only objects and `null` are covered.

### CODE REVIEW

**Comment by MrWolfZ** [comment]
@tbroadley sorry for the late response, I was moving to a new apartment the last few days. The PR looks good. While the test object you use already checks most types I agree it would be good to have tests that check the conversion for non-nested values of all types you mentioned. In addition I noticed that the second direction of the double conversion tests is missing (i.e. '[...]state to view and back') which I indeed forgot for the dateToISOString converter myself. Once those tests are added I am happy to merge this PR.

PS: thanks for fixing the whitespace issues in the README. However, in the future I recommend making such changes in a separate PR so that only the code that is really affected by your feature appears in the changelog.

**Comment by MrWolfZ** [comment]
Oh, and also, I have introduced a `develop` branch so that the `master` is always in sync with the latest released version of the library. Please re-target your PR to `develop`.

**Comment by tbroadley** [comment]
@MrWolfZ Thank you for the feedback! I'll add the tests that you've suggested.

File: src/control/value-converter.spec.ts [hunk 1]
```diff
@@ -41,5 +41,78 @@ describe('NgrxValueConverters', () => {
...
+          a: [1, 2, 3],
+          b: {
+            c: '456'
+          },
+        },
+        expectedStateValue: '{"a":[1,2,3],"b":{"c":"456"}}',
+      },
+    ];
+
+    tests.forEach(({ type, expectedStateValue, expectedViewValue }) => {
```
Comment (tbroadley) [inline src/control/value-converter.spec.ts]: Iterate over each type (string, number, boolean, array, and object) and check that the following conversions are correct:

- View to state
- State to view
- View to state to view
- State to view to state

**Comment by MrWolfZ** [comment]
Thanks for the great work.

**Comment by tbroadley** [comment]
Thank you!

**Pull request merged**